File: vue/ps5GameTradingSystem/src/types/shoppingCar.ts
export interface shoppingCar {
    id:string
    userId:string
    gameName:string
    quantity:number
    price:number
}
export interface shoppingCarItem {
    id: string        // 对应后端的 id
    userId:string
    gameName: string      // 对应后端的 gameName
    quantity:number
    price:number
    checked: boolean  // 前端专属的选中状态
}
export type shoppingCars = shoppingCar[];
export type shoppingCarItems = shoppingCarItem[];

File: springboot/ps5GameTradingSystem/src/main/java/com/zxy/ps5gametradingsystem/entity/Shoppingcar.java
package com.zxy.ps5gametradingsystem.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("`shoppingcar`")
public class Shoppingcar {
    private  String id;
    private  String userId;
    private  String gameName;
    private  Integer quantity;
    private  Float price;
}
